services/wallet/router/sendtype/send_type.go

```diff
@@ -39,8 +39,8 @@ func (s SendType) IsStickersTransfer() bool {
 	return s == StickersBuy
 }
 
-func (s SendType) FetchPrices(marketManager *market.Manager, tokenID string) (map[string]float64, error) {
-	symbols := []string{tokenID, "ETH"}
+func (s SendType) FetchPrices(marketManager *market.Manager, tokenIDs []string) (map[string]float64, error) {
+	symbols := append(tokenIDs, "ETH")
```

Comment:
What if `tokenIDs` already contains "ETH"?
I guess `symbols ` should have unique tokens at the end.

Also, not sure if needed the "if" below if you add this.

Reply:
`I guess symbols should have unique tokens at the end.`
aaah, I will deduplicate this seq!

```
if s.IsCollectiblesTransfer() {
		symbols = []string{"ETH"}
	}
```

This will still be needed because we reset it to only for ETH?

Reply:
> This will still be needed because we reset it to only for ETH?

correct

Reply:
done @saledjenic